rest-server/spec/factories/meetups.rb

```diff
@@ -22,6 +22,19 @@
     end_at   { start_at + 1.hour }
 
     association :group, factory: :group
-    association :venue, factory: :venue
+
+    trait :physical_venue do
+      after(:create) do |meetup|
+        physical_venue = create(:physical_venue, group: meetup.group)
+        meetup.update_attribute(:venue_id, physical_venue.id)
```

Comment:
__[RuboCop]__

Rails/SkipsModelValidations: Avoid using `update_attribute` because it skips validations.


<!-- Comment by [SideCI](https://sideci.com) -->